### DISCUSSION TITLE
It would make more sense if the Maps Stats Page listed the maps, then the win-rate for each hero, sorted by best.

### DISCUSSION THREAD

**Discussion by pizzaisdavid** [Heroes-Profile/heroesprofile#1065] [General]
https://www.heroesprofile.com/Global/Hero/Maps

Usecase; I am a noob, I want to pick a hero that is good for Cursed Hollow. I want to go to maps, click Cursed Hollow, then see a list of the best heroes.

Optional: be able to filter by role.

**Comment by Zemill** [comment]
Would probably be better to just filter on map at 

https://www.heroesprofile.com/Global/Hero

**Reply by pizzaisdavid** [reply]
nice, that fits my requirements, thanks
